app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+class Error(Exception):
+    pass
+
+
+class FileNameError(Error):
+    pass
+
+
+class CommandError(Error):
+    pass
+
+
+def copy_file(command: str) -> None:
+    first_arg, second_arg, third_arg = command.split()
+    if first_arg != "cp":
+        raise CommandError("You privided the wrong command")
+    if second_arg == third_arg:
+        raise FileNameError("File names can not be equal")
```

Comment:
Don't raise an exception here. You need to finish execution of function if parameter isn't correct.
https://github.com/mate-academy/py-copy-file/tree/master#:~:text=It%20must%20do%20nothing%20in%20case%20the%20user%20is%20trying%20to%20copy%20file%20to%20file%20with%20the%20same%20name.